Networking/Networking/Remote/WooShippingRemote.swift

```diff
@@ -120,13 +122,36 @@ public final class WooShippingRemote: Remote, WooShippingRemoteProtocol {
             completion(.failure(error))
         }
     }
+
+    /// Loads account settings.
+    /// - Parameters:
+    ///   - siteID: Remote ID of the site.
+    ///   - completion: Closure to be executed upon completion.
+    public func loadAccountSettings(siteID: Int64,
+                                    completion: @escaping (Result<WooShippingAccountSettingsResponse, Error>) -> Void) {
+        do {
+            let path = Path.accountSettings
+            let request = JetpackRequest(wooApiVersion: .wooShipping,
+                                         method: .get,
+                                         siteID: siteID,
+                                         path: path,
+                                         availableAsRESTRequest: true)
+
+            let mapper = WooShippingAccountSettingsMapper(siteID: siteID)
+
+            enqueue(request, mapper: mapper, completion: completion)
+        } catch {
+            completion(.failure(error))
+        }
```

Comment:
Ah, just a minor thing but the `do`/`catch` isn't needed here because there's nothing that throws in the `do` block.